### PR TITLE
Creating project

### PR DESCRIPTION
**Pull request by Dziarnek**

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+var treeLayers = 0;
+
+function drawTree(layers) {
+    for (var i = 0 ; i < layers ; i++ ) {
```
Comment (matwjablonski) [inline js/scripts.js]: remove space before `;`

File: js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        var star = "*";
+        for (var j = 0 ; j < i ; j++) {
```
Comment (matwjablonski) [inline js/scripts.js]: remove space before `;`

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/scripts.js (hunk 1 above)
Comment (Dziarnek) [reply js/scripts.js]: Done :)

File: js/scripts.js (hunk 2 above)
Comment (Dziarnek) [reply js/scripts.js]: Done :)

**Pull request merged**